packages/server/src/api/notification/notification.router.ts

```diff
@@ -50,4 +51,39 @@ router.put(
   controller.updateUserNotificationSetting,
 );
 
+router.get(
+  '/:orgId/users-requesting-medical-attention',
+  bodyParser.json(),
+  requireOrgAccess,
```

Comment:
These should probably all have `requireInternalUser` in addition to `requireOrgAccess` to make sure only internal applications are able to use this endpoint.

Reply:
Ah ok this makes sense, thanks!